src/core/messages-reactions.ts

```diff
@@ -207,7 +207,9 @@ export class DefaultMessageReactions implements MessagesReactions {
       return;
     }
     if (!event.summary) {
-      return;
+      // This means the summary is now empty, which is valid.
+      // Happens when there are no reactions such as after deleting the last reaction.
+      event.summary = {};
```

Comment:
Is there a test case that covers this behaviour (I assume not?) - if not can we add one please

Reply:
Added